AutoMapper.Data.Tests/DataReaderMapping.cs

```diff
@@ -614,6 +620,35 @@ public void Then_results_should_be_as_expected()
         private IMapper _mapper;
     }
 
+   public class When_mapping_a_data_reader_to_a_dto_with_an_interface_member
+    {
+        public interface INoneSuch
+        {
+            string Nop();
+        }
+
+        public class NoneSuch
+        {
+            public string name { get; set; }
+            public INoneSuch value { get; set; }

```

Comment:
What about adding these two to the existing test and asserting what the values should be after map? I would guess it's the default values, but it's more obvious to see the assert.